2019-06-21-org-wide-pm.md

```diff
@@ -0,0 +1,102 @@
+- garry:
+	- what are the things we want to **follow through** on in the next few months?
+    - the things we experiment with are **not commitments forever**
+    - things we experiment with, we should be able to move away
+    - the things we’re experimenting with organizational values and objectives
+    - example:
+        - we want to try trello
+        - exportability. lock-in escape.
+        - we should be able to take a risk. we have little to lose on choosing something like trello.
+        - reminds of **value proposition** (from Pivotal): that products can work on mainstream clouds or your own. recognizing many diff types of business and many diff types of challenges.
+- patcon: **open source tools are NOT always the best ones**. our asset is **trustworthiness**, and sometimes we should introduce closed source tools if they t
+- I’d like to have a process making choices on client behalf that are **grounded in reality of what's succeeding in the world, and why?** we should keep knitting together the tools we want to win, with the ones that are common (for many good reasons) in this current moment.
+- scrappy nonprofits sometimes just need to **build political will** with a **trusted partner**. if we’re pushing something that adds to their **burden** (which they maybe can’t afford to tackle yet), then we’re perhaps **abusing that trust**
+- responsibilties
+	- actions in task tracker within one day of all-hands meeting
+	- agenda published x days before meetings
+	- clean-up within x days
+	- who reviews? when do we merge into archive?
+	- #todo ask in meeting about others’ very specific core activities missing
+	- #todo create loomio thread before all-hands
+- patcon re: **pol.is workshop**
+	- garry: trying to be pragmatic. wondering **how we start** the engagement. what trust building comes first before they invite us in.
+	    - patcon: backlog of connections/opportunities from **civic tech and CSI space**.
+	    - not sure **how it fits into our offerings.**
+	    - how does it serve us?
+    - patcon: feeling it's perfect. be entrusted to come into complex blockage. engage in freeform process where all tensions are surfaced among large group. be the hero who helps them have better conversations and work through.
+	        - following a **successful session**:
+	            - by design, went into **deeply emotional topics** of organization
+	            - gained **trust for helping** them navigate situation that felt intractible
+	            - gain deep **knowledge of challenges** from outputs of process
+	            - surprise in unexpected technological solution to soft human challenge. **open mind** to hear more.
+
+## Action Items
+
+- [x] Review and clean notes (patcon+garry)
+- [x] Double-check with Garry whether we want to arrive with a draft doc (patcon)
+    - We don't. Go in for convo.
+- [x] ~~Write draft role proposal~~
+- [ ] Prepare summary in agenda
```

Comment:
Are we good to merge or are we waiting on this item?

Reply:
We're good -- bolding served that purpose

Reply:
```suggestion
- [x] Prepare summary in agenda. (bolding accomplished this)
```